lesson_13/json_task.py

```diff
@@ -0,0 +1,34 @@
+import logging
+import glob
+import json
+from fileinput import filename
+from pathlib import Path
+
+"""Завдання 2:
+Провалідуйте, чи усі файли у папці ideas_for_test/work_with_json є валідними json. результат для 
+невалідного файлу виведіть через логер на рівні еррор у файл json__<your_second_name>.log
+"""
+
+files_json = glob.glob(r'/lesson_13/work_with_json/*.json')
+#Variable with path to files
+
+
+def find_json_files(files_json):
+    logging.basicConfig(
+        filename='../lesson_13/json_nazarenko.log',
+        level=logging.ERROR,
+        format='%(asctime)s - %(message)s')
+
+    try:
+        for file in files_json:
+            with open(file, 'rb') as f:
+                json.load(f)
+            #print("ok")
+    except ValueError as e:
```

Comment:
According Doc [Json Load](https://docs.python.org/3/library/json.html#json.load) here should be special error [JSONDecodeError](https://docs.python.org/3/library/json.html#json.JSONDecodeError)